adapters/faraday/lib/opentelemetry/adapters/faraday.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Faraday
+      module_function
+
+      TRACER_NAME = 'faraday'
+
+      def install(config = {name: TRACER_NAME,
+                            version: tracer_version})
+        require_relative 'faraday/adapter'
+        Faraday::Adapter.install(config)
+      end
+
+      def tracer_version
+        Gem.loaded_specs[TRACER_NAME]&.version.to_s
```

Comment:
If the `name` is overridable, the `version` is looked up, potentially, using a different name from the one provided. Is that the right thing to do? The `version` returned here would be the Faraday version rather than the adapter gem version. I'm not sure that's the intent -- I *think* the version is supposed to be the adapter version.

Reply:
We discussed this at our last SIG - the version should be the version of the adapter and we've removed it from the config and made it so it is not overridable.